### ISSUE TITLE
FilterOperator.IN

### ISSUE THREAD

**Issue by lorenzleutgeb** [crnk-project/crnk-framework#497]
I'd like to have `FilterOperator.IN` with semantics similar to SQL's `IN` operator. To the left I expect some `PathSpec` and to the right a set of values.

**Comment by remmeier** [comment]
yes would make sense to have. `FilterOperator.IN` is easy to add. The other place would be `JpaCriteriaQueryBackend.handle(...)`. There actually such a functionality is already in place, but so far not properly modelled through an operator.

**Comment by lorenzleutgeb** [comment]
On a second thought, I also would like to have a set on the left and a `PathSpec` on the right, to model something like: `'betty' IN friends.name' (where `friends` refers to a relation on some resource, and the related resource has an attribute called `name`).

**Comment by remmeier** [comment]
some suggestions from the json:api spec would ve good in the area. so far there is no support for evaluated values, but not much of preventing it. altough so far in my cases i settled with additional attributes, resources, relationship and alsways attempted to keep the resources and queries simple
